src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+import { PeopleInfo } from '../PeopleInfo';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
use interface prefer. 
Types have unique features but the interface is best for scaling.